daemon/api_snaps.go

```diff
@@ -203,6 +203,7 @@ type snapInstruction struct {
 	Snaps                  []string               `json:"snaps"`
 	Users                  []string               `json:"users"`
 	ValidationSets         []string               `json:"validation-sets"`
+	QuotaGroupName         string                 `json:"quota-group"`
```

Comment:
validate() should check that this is set only when it makes sense?

Reply:
I added one that checks for the option being present only for install command